scripts/vote_2023_10_03.py

```diff
@@ -0,0 +1,70 @@
+"""
+Voting 03/10/2023.
+
+1. Update Anchor Vault implementation from 0x07BE9BB2B1789b8F5B2f9345F18378A8B036A171 to #TBA
+2. Finalize Anchor Vault upgrade
+
+Vote passed & executed on #, block #
+"""
+
+import time
+
+from typing import Dict, Tuple, Optional
+
+from brownie.network.transaction import TransactionReceipt
+from utils.voting import bake_vote_items, confirm_vote_script, create_vote
+from utils.agent import agent_forward
+
+from utils.config import (
+    get_deployer_account,
+    contracts,
+    get_is_live,
+    get_priority_fee,
+)
+
+ANCHOR_NEW_ADDRESS = "#TBA"
+
+def start_vote(tx_params: Dict[str, str], silent: bool = False) -> Tuple[int, Optional[TransactionReceipt]]:
```

Comment:
should have named `start_votes` maybe cause returns a tuple of votes :thinking:

Reply:
no, it's a one vote which returns tuple of data

Reply:
Yeah, I see, but we still return a single vote wrapped into a list :)
```return confirm_vote_script(vote_items, silent) and list(create_vote(vote_items, tx_params))```

not have a strong opinion here; can resolve as you wish :pray: